src/test/pybind/test_rados.py

```diff
@@ -333,6 +333,12 @@ def test_trunc(self):
         size = self.ioctx.stat('abc')[0]
         eq(size, 2)
 
+    def test_cmpext(self):
+        self.ioctx.write('test_object', b'abcdefghi')
+        eq(0, self.ioctx.cmpext('test_object', b'abcdefghi', 0))
```

Comment:
think we can check for None if the function runs successfully if we update with an assert.